.circleci/ci_increase_chart_version.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+"""
+This is the script that would send a dispatch event to the helm chart repo to auto increase chart version.
+You can set the UPDATE_DEV flag to decide whether to update to dev too.
+For master, we increase the chart version and update dev together. The app version should be short git sha with length 7.
+For release, we increase the chart version only. The app version should be semver. (eg. 1.0.3-pre.0)
+Required env vars:
+- CHART_NAME (eg. webwallet)
+- APP_VERSION (eg. 3d75118)
+- HELM_CHART_REPO (eg. helm-devlopement)
+- UPDATE_DEV (true/false)
+- GITHUB_API_TOKEN
+"""
+
+set -ex
+
+[ -z "$CHART_NAME" ] && echo "CHART_NAME should be set" && exit 1
+[ -z "$APP_VERSION" ] && echo "APP_VERSION should be set" && exit 1
+[ -z "$HELM_CHART_REPO" ] && echo "HELM_CHART_REPO should be set" && exit 1
+[ -z "$UPDATE_DEV" ] && echo "HELM_CHART_REPO should be set" && exit 1
```

Comment:
typo here, should be `UPDATE_DEV should be set`

Reply:
nice catch